### ISSUE TITLE
RAS syndrome and RAS acronym syndrome 

### ISSUE THREAD

**Issue by Franck-Dernoncourt** [mimic2/v3.0#46]
Just a stylistic note: I see a lot of [RAS syndromes](http://en.wikipedia.org/wiki/RAS_syndrome) in the documentation, [e.g.](http://mimic2.github.io/v3.0/dat/icustayevents.html):

> For patients transferred out of **ICU units** but re-admitted to the same or different **ICU care unit** within 24 hours, it is considered as one ICUSTAY event with the same ```ICUSTAY_ID```. However, if the patient was re-admitted back to the same or different **ICU care unit** after 24 hours, it is considered as a new ICUSTAY event and is assigned a new ```ICUSTAY_ID```**.

Do we want to clean them?

**Comment by tompollard** [comment]
Yep, we should clean this up. There are a bunch of changes that need to be made to the documentation, so I'll make sure this is included. Pull requests welcome in the meantime :)
